python/google/protobuf/pyext/message.cc

```diff
@@ -1921,6 +1922,9 @@ static PyObject* MergeFromString(CMessage* self, PyObject* arg) {
   AssureWritable(self);
   io::CodedInputStream input(
       reinterpret_cast<const uint8*>(data), data_length);
+#if ALLOW_UNLIMITED_BINARY_PROTO
```

Comment:
Please make this symbol include "PYTHON", eg. `PROTOBUF_PYTHON_ALLOW_OVERSIZE_PROTOS`.